docs/source/gen/flytectl_delete_execution-queue-attribute.rst

```diff
@@ -10,15 +10,15 @@ Synopsis
 
 
 
-Deletes execution queue attributes for the given project and domain combination or additionally with workflow name.
+Delete execution queue attributes for the given project and domain combination or additionally with workflow name.
 
 For project flytectldemo and development domain, it is:
 ::
 
  flytectl delete execution-queue-attribute -p flytectldemo -d development 
 
 
-Deletes execution queue attribute using config file which was used for creating it.
+Delete execution queue attribute using config file which was used for creating it.
 Here, the config file is written to era.yaml.
 Value is optional in the file as it is unread during the delete command but it can be kept since the same file can be used for get, update or delete commands. 
```

Comment:
```suggestion
Value is optional in the file as it is unread during the delete command, but it can be kept since the same file can be used for get, update or delete commands. 
```